client/deprecated-hid-flasher/flasher/usb_cmd.h

```diff
@@ -12,16 +12,8 @@
 
 #ifndef __USB_CMD_H
 #define __USB_CMD_H
-#ifdef _MSC_VER
-typedef DWORD uint32_t;
-typedef BYTE uint8_t;
-#define PACKED
-// stuff
-#else
-#include <stdint.h>
-#include <stdbool.h>
-#define PACKED __attribute__((packed))
-#endif
+
+#include "common.h"
```

Comment:
client/deprecated-hid-flasher/flasher/usb_cmd.h should be left untouched because it's standalone without access to pm3 include files.
Or you can add the definition of PACKED in its client/deprecated-hid-flasher/flasher/proxmark3.h

Reply:
btw. it cant compile

Reply:
unknown type name 'PVOID'
unknown type name 'UCHAR'
etc...

Reply:
stuff from windows only.

Reply:
may you check?

Reply:
client/deprecated-hid-flasher/flasher compiles fine under Linux.
Feel free to fix compilation under Windows but don't worry too much, it's deprecated ;)

Reply:
ok, let it be in this state...

Reply:
yeah,  don't use it.  If the ppl want to upgrade from HID -> CDC they have plenty of JTAG options today.  I would say remove it completely